metatron-app/metatron-discovery
# Fn fix ifcountd error

### Description
Fixed an error that occurred when using the `ifcountd` function in a user-defined field.

**Related Issue** : None

### How Has This Been Tested?
1. Create a dashboard utilizing the Sales data source.
2. Create a user-defined column of measure type.
3. Create a expression using the `ifcountd` function, enter an arbitrary column name, and create a column. ex. `IFCOUNTD( [Region]  != 'East',  [Region] )` 
![image](https://user-images.githubusercontent.com/822255/65010031-978dbf80-d949-11e9-9a03-34ae7f436f0e.png)
4. Create a chart using this column.
5. Confirm that the chart is displayed without errors.
![image](https://user-images.githubusercontent.com/822255/65010299-59dd6680-d94a-11e9-9ab5-7106e703ba73.png)

#### Need additional checks?
- Check that no other aggregate functions are working.

### Types of changes



### Checklist:


### Additional Context
